recipe/meta.yaml

```diff
@@ -8,16 +8,16 @@ package:
 
 source:
   - url: https://github.com/earth-system-radiation/pyRTE-RRTMGP/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: b161ac8b1ddca59eb994eb44ba8aabb9a596f50f24a8fd0886fea8c680e41a29
+    sha256: b53de2a06648f3bdd719dfea07d31fec7f266519a4de5a2028535c4ecf8f9808
   - url: https://github.com/earth-system-radiation/rte-rrtmgp/archive/refs/tags/v{{ rte_rrtmgp_version }}.tar.gz
     fn: rte-rrtmgp-{{ rte_rrtmgp_version }}.tar.gz
     sha256: e3830007529a3fb35727683ff990cac7e6792cdee4c9d9b07a55c969a97c95e5
     folder: rte-rrtmgp
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
-  number: 2
-  skip: True  # [not linux]
+  number: 0
+  skip: true  # [not linux]
```

Comment:
@giancastro The Fortran libraries are now distributed also for macOS (intel and apple silicon). Can we also distribute `conda` packages for those platforms?